src/style.css

```diff
@@ -1,3 +1,65 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+:root {
+  --main-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__link {
+  text-decoration: none;
+  font-size: 12px;
+  text-transform: uppercase;
+  color: #000;
+  display: flex;
+  height: 60px;
+  line-height: 60px;
+}
+
+.nav__item:not(:first-child) {
```

Comment:
It would be better to use margin-left instead of padding-left to separate nav items. Padding can cause unwanted changes in the box size.